src/frontend/src/eth/components/receive/EthReceive.svelte

```diff
@@ -23,5 +25,11 @@
 </script>
 
 <ReceiveButtonWithModal open={openReceive} isOpen={$modalEthReceive}>
-	<EthReceiveModal slot="modal" />
+	<ReceiveModal slot="modal" address={$networkAddress ?? undefined}>
+		<svelte:fragment slot="content">
+			{#if $networkEthereum}
+				<EthReceiveMetamask />
```

Comment:
out of curiosity: why the `slot` definition is not here but outside the `if` condition?

Reply:
This is required by Svelte (`Svelte: <svelte:fragment> must be a child of a component`). More discussions here - https://github.com/sveltejs/svelte/issues/5604.

Reply:
wow! thank you!